Mathlib/NumberTheory/Bernoulli.lean

```diff
@@ -152,9 +152,7 @@ theorem bernoulli'PowerSeries_mul_exp_sub_one :
   simp_rw [mem_antidiagonal]
   rintro ⟨i, j⟩ rfl
   have := factorial_mul_factorial_dvd_factorial_add i j
-  field_simp [mul_comm _ (bernoulli' i), mul_assoc, add_choose]
-  norm_cast
-  simp [mul_comm (j + 1)]
+  simp (disch := assumption) [field, add_choose]
```

Comment:
```suggestion
  simp_all [field, add_choose]
```

I'd prefer we only use custom dischargers if standard tactics don't work. It feels like unnecessary optimization to use `disch := assumption` over `simp_all` or `simp [*]`.